src/main/java/persistence/entity/persistentcontext/JdbcEntityPersister.java

```diff
@@ -25,8 +24,8 @@ public class JdbcEntityPersister<T> implements EntityPersister<T> {
   public JdbcEntityPersister(Class<T> clazz, JdbcTemplate jdbcTemplate, MetaEntity<T> metaEntity) {
```

Comment:
```suggestion
  public JdbcEntityPersister(JdbcTemplate jdbcTemplate, MetaEntity<T> metaEntity) {
```

사용하지 않는 매개변수는 필요없을 것 같네요

Reply:
네!~ 삭제 했습니다!!